src/ui/zcl_abapgit_services_git.clas.abap

```diff
@@ -228,7 +228,8 @@ CLASS zcl_abapgit_services_git IMPLEMENTATION.
           lv_answer                 TYPE c LENGTH 1,
           lt_unnecessary_local_objs TYPE zif_abapgit_definitions=>ty_tadir_tt,
           lt_selected               LIKE lt_unnecessary_local_objs,
-          lt_columns                TYPE stringtab.
+          lt_columns                TYPE stringtab,
+          ls_checks                 TYPE zif_abapgit_definitions=>ty_deserialize_checks.
```

Comment:
suggest defining a new type instead of reusing `ty_deserialize_checks`, as the deletion and delete_checks contains different logic than for deserialization